anvio/migrations/profile/v35_to_v36.py

```diff
@@ -0,0 +1,177 @@
+
+            for view in essential_data_fields_for_anvio_profiles:
+                table_name = f'{view}_{target}'
+
+                # le creationeaux au de neuvo tabl
+                profile_db._exec(f'''CREATE TABLE {table_name} (item text, layer text, value numeric)''')
+                view_data = []
+                for split_name in atomic_data:
+                    view_data.append((split_name, sample_name, atomic_data[split_name][view]), )
+
+                # populate the new view table
+                profile_db._exec_many('''INSERT INTO %s VALUES (?,?,?)''' % (table_name), view_data)
+
+                # update the views table
+                if target == 'splits':
+                    profile_db._exec('''INSERT INTO views VALUES (?,?)''', (view, table_name))
+
+        # баяртай
+        profile_db.disconnect()
+
+    elif is_full_profile and is_merged:
+        #########################
+        #     MERGED PROFILE    #
+        #########################
+
+        # open the profile database without rowid prepend.
+        profile_db = db.DB(db_path, None, ignore_version = True, skip_rowid_prepend=True)
+
+        # learn your samples
+        sample_names = [s.strip() for s in profile_db.get_meta_value('samples').split(',')]
+
+        # drop the contents of the view table.
+        profile_db._exec("DELETE FROM views")
+
+        for target in ['splits', 'contigs']:
+            for view in essential_data_fields_for_anvio_profiles:
+                table_name = f'{view}_{target}'
+
+                progress.update(f"Working on table '{table_name} ...'")
+
+                table_data = profile_db.get_table_as_dict(table_name)
+
+                # drop the old view table
+                profile_db._exec(f'DROP TABLE {table_name}')
+
+                # create a new view table!
+                profile_db._exec(f'''CREATE TABLE {table_name} (item text, layer text, value numeric)''')
+
+                # fill in the new view data from the old format
+                view_data = []
+                for split_name in table_data:
+                    for sample_name in sample_names:
+                        view_data.append((split_name, sample_name, table_data[split_name][sample_name]), )
+
+                # populate new view table
+                profile_db._exec_many('''INSERT INTO %s VALUES (?,?,?)''' % (table_name), view_data)
+
+                # if splits, I sits
+                if target == 'splits':
+                    profile_db._exec('''INSERT INTO views VALUES (?,?)''', (view, table_name))
+
+        # さようなら
+        profile_db.disconnect()
+
+    else:
+        ###########################
+        #     SURPRISE PROFILE    #
+        ###########################
+
+        raise ConfigError("Anvi'o is confuse. Your profile database does not fit into anything we have "
+                          "anticipated to run into here. For full disclosure, ")
```

Comment:
I think this sentence needs an end :p

Reply:
lol

Reply:
5a775d1724654ec8ff623e309718b79d73782082 addresses this. I will merge this repo soon to make sure we start testing it.

Reply:
yay :)

btw, this migration script is refreshingly full of easter eggs and it makes me very happy :)
> баяртай
>さようなら
> if splits, I sits

things like this make code so much more enjoyable to read ❤️

Reply:
hehe :) I'm glad that someone is reading the code :)